### PR TITLE
Use dataVolumeTemplates instead of creating PVC/DV objects.

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,9 +13,9 @@
     "/dist"
   ],
   "scripts": {
-    "lint": "yarn lint:js:browser && yarn lint:js:node && yarn lint:sass",
-    "lint:js:browser": "eslint src --config config/eslint.browser.js",
-    "lint:js:node": "eslint config tools --config config/eslint.node.js",
+    "lint": "yarn lint:js:browser && yarn lint:js:node && yarn lint:sass --fix",
```
Comment (mareklibra) [inline package.json]: This change definitely needs to go to a separate PR.

Anyway, I am not happy with this approach. What about building at release time? We should not change sources there.

I think, we should add new `lint:fix` script handling that.

Comment (rawagner) [reply package.json]: I actually forgot to remove it, it shouldn't be here :) 

But I agree to add new lint:fix script and create a separate PR for that

Comment (rawagner) [reply package.json]: https://github.com/kubevirt/web-ui-components/issues/211

**Comment by rawagner** [comment]
@mareklibra this PR is ready for review :)

File: src/components/Wizard/CreateVmWizard/fixtures/ResultTab.fixture.js [hunk 2]
```diff
@@ -14,15 +14,15 @@ export default [
     name: 'Error',
     props: {
       success: false,
-      result: 'Failed',
+      result: ['Failed'],
```
[the remark this answers is not in the document]
Comment (rawagner) [reply src/components/Wizard/CreateVmWizard/fixtures/ResultTab.fixture.js]: fixed
